src/_includes/news.md

```diff
@@ -0,0 +1,37 @@
+{% assign releases = site.documents %}
+{% assign releases = releases | sort: "release_date" | reverse %}
+
+    <h3>Recent Releases</h3>
+    <div class="row align-middle">
+
+{% assign collections = "" %}
+{% assign i = 0 %}
+
+{% for release in releases %}
+    {% if i > 2 %}
+        {% break %}
+    {% endif %} 
+    {% if collections contains release.collection  %}
+        {% continue %}
+    {% else %}
+        {% assign collections = collections | append:'|' | append:release.collection %}
+    {% endif %}
+    <div class="col-lg-4 col-md-auto">
+        <div class="card card-orange">
+            <div class="card-body">
+                {% if release.collection contains "artemis" %}
+                    <h5 class="card-title text-blue">Artemis {{release.version}} Release</h5>
+                    <a style="margin-top: 10px" href="./components/artemis/download/past_releases" class="btn btn-secondary float-right">Release<i class="fa-download fa btn-icon"></i></a>
```

Comment:
This should link to the download page. The past/previous releases (as in not-current releases) page doesnt actually contain the release the entry is about.